content/docs/2_reference/6_system/1_options/0_panel/panel-menu/reference-article.txt

```diff
@@ -0,0 +1,143 @@
+Title: panel.menu
+
+----
+
+Intro: Option to customize the Panel menu
+
+----
+
+Text:
+
+
+<since v="4.0.0">
```

Comment:
Would it make more sense to use the Since field for it?

Reply:
Oh 100%. Had it first not as separate article and that's why I wrapped it. Then just moved it as is.